src/Models/BaseElement.php

```diff
@@ -5,7 +5,9 @@
 use DNADesign\Elemental\Controllers\ElementController;
 use DNADesign\Elemental\Forms\TextCheckboxGroupField;
 use DNADesign\Elemental\ORM\FieldType\DBObjectType;
+use DNADesign\Elemental\TopPage\TopPageTrait;
 use Exception;
+use Page;
```

Comment:
```suggestion
use SiteTree;
```
I don't like using the `Page` class here as it lives at the Project level and it can have literally anything on it.  We should be using the `SiteTree` class here instead

Reply:
I think there was a wider discussion started somewhere whether `SiteTree` and `Page` should become one again (and be extended via extensions or projects' own `BasePage` sort of thing) because of modules requiring/using `Page` class which is defined in the project space.

Until that is decided I think either approach works fine as this is not the only instance where that would need changing. Or are we (semi-/officially) suggesting that everywhere a relation to a page is needed within the SS modules' area it should use SiteTree class to be universal? A lot of things would not function without the `Page` class existing anyway.